apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/AplAppStatus.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright © 2018 Apollo Foundation
+ */
+package com.apollocurrency.aplwallet.apl.core.app;
+
+import com.apollocurrency.aplwallet.api.dto.DurableTaskInfo;
+import com.apollocurrency.aplwallet.apl.util.StringUtils;
+import java.text.DecimalFormat;
+import java.text.NumberFormat;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import org.slf4j.Logger;
+import static org.slf4j.LoggerFactory.getLogger;
+
+/**
+ *
+ * @author alukin@gmail.com
+ */
+@Singleton
+public class AplAppStatus {
+    private static final Logger LOG = getLogger(AplAppStatus.class);
+    private static final long ONE_DAY=3600*24;
+    private final Map<String,DurableTaskInfo> tasks = new HashMap<>();
+    private NumberFormat formatter = new DecimalFormat("#000.000");
+    
+    @Inject
```

Comment:
Nothing to inject here

Reply:
Well, yes but I'd rather indicate that this is injectable entity